### ISSUE TITLE
status_of_proc does not exist in Red Hatish environments

### ISSUE THREAD

**Issue by sdherr** [neo4j/neo4j#10067]
Your RPM build (much appreciated by the way!) has a bug in the SysVinit script.

In do_status() it calls "status_of_proc". That's a Debian tool that does not exist on Red Hatish systems.

$ service neo4j status
/etc/init.d/neo4j: line 93: status_of_proc: command not found

**Comment by drbayer** [comment]
I ran into the same problem.  This occurs only if you're running on RHEL derivatives and have LSB installed.  I was able to update the has_lsb_init() function as follows to make it work & still be cross-platform compatible:
```
has_lsb_init()
{
  test -f "/lib/lsb/init-functions" && [ $(grep -c status_of_proc /lib/lsb/init-functions ) -gt 0 ]
}
```

**Comment by davidlrosenblum** [comment]
I ran into this when I had to install Neo4j on an VM where LSB things were installed for other packages  that needed to run.   Since I was on a Red Hat related distro, I did not want to use the LSB scripts, so I just fooled the script into thinking that LSB is not installed by changing line 59 from
test -f "/lib/lsb/init-functions"
to
test -f "/lib/lsb/init-functionsx"
so it wouldn't find the LSB scripts and behave as if they aren't there.  Ok for a testing things out.
